### PR TITLE
4.8.2

### PR DESCRIPTION
**Pull request by iElden**

### CODE REVIEW

**Comment by iElden** [comment]
Boot test : OK

[BUG] : Unique Spad not at 5 Iron
image

**Comment by iElden** [comment]
LOC OK:
#174
#192 (automatically updated by the game)
#184
#180
#188 (automatically updated by the game)
#195 (No LOC)

LOC MISSING (to check) : 
#191
#178
#190 (Miss English, only Fr was updated)
#189
#193
#194
#185 (Miss English, only Fr was updated)

File: sql/base.sql [hunk 1]
```diff
@@ -1076,7 +1079,7 @@ INSERT OR IGNORE INTO RequirementArguments (RequirementId , Name , Value)
 -- Great Library unlocks at Drama & Poetry instead of Recorded History
 UPDATE Buildings SET PrereqCivic='CIVIC_DRAMA_POETRY' WHERE BuildingType='BUILDING_GREAT_LIBRARY';
 
--- Venetian Arsenal gives 100% production boost to all naval units in all cities instead of an extra naval unit in its city each time you build one
+-- Venetian Arsenal gives 80% production boost to all naval units in all cities instead of an extra naval unit in its city each time you build one
```
Comment (ZhenjaMax) [inline sql/base.sql:1082]: Why comment 80% but coded 75% ?

**Comment by iElden** [comment]
#197 All English LOC : OK

**Comment by iElden** [comment]
Boot test FAIL
```
[555700.155] [Localization] ERROR: Database::XMLSerializer (spanish.xml): Duplicate column names detected in <Row>. Name: Replace, Value: 
[555700.155] [Localization]: In XMLSerializer while updating table LocalizedText from file spanish.xml.
```

**Comment by iElden** [comment]
Boot test : OK

ERROR WITH LOCs, NON ASCII TEXT DISPLAYED !!!

**Pull request merged**